src/python/WMCore/WorkQueue/DataStructs/WorkQueueElementsSummary.py

```diff
@@ -65,3 +78,29 @@ def getWQElementResultsByReauest(self, requestName = None):
             return self.wqResultsByRequest.get(requestName, None)
         else:
             return self.wqResultsByRequest
+    
+    def printSummary(self, request):
+        
+        wqResult = self.getWQElementResultsByReauest(request)
```

Comment:
Just because it's consistent among several places, it does not mean it has no typo :-D
"getWQElementResultsByReauest" read "getWQElementResultsByRequest"